### ISSUE TITLE
aria-labelled-by should be aria-labelledby

### ISSUE THREAD

**Issue by anthonator** [rackt/react-a11y#87]
According to the [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute) `aria-labelled-by` should be `aria-labelledby`. I haven't found any mention on the web that `aria-labelled-by` is a thing. Even if it is you should also support `aria-labelledby`.

**Comment by anthonator** [comment]
:+1: Thanks!
